src/utilities.js

```diff
@@ -25,18 +34,29 @@ const getLanguageCode = () => {
   return languageCode;
 };
 
+const getCookieDomain = () => {
+  if (isLocalhost()) {
+    return LOCALHOST;
+  }
+
+  return `.${firstMatchingStageEnvironment() || 'edx.org'}`;
+};
+
 const createHasViewedCookieBanner = () => {
   const path = '/';
 
   // Setting maxAge to 2^31 -1
   // because Number.SAFE_MAX_INTEGER does not get processed properly by the browser
   // nor does the max Date defined in http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
   const maxAge = 2147483647;
+  const domain = getCookieDomain();
```

Comment:
Nitpick: These 3 ```const```s should be within the if statement because there's no point defining them if they don't get used. And I guess not all of them need to be variables.